LocalSettings.php

```diff
@@ -6765,8 +6765,7 @@
 	'evelution' => 'Incompatible with MediaWiki 1.42',
 	'eveskin' => 'Incompatible with MediaWiki 1.42',
 	'femiwiki' => 'Incompatible with MediaWiki 1.42',
-	'snapwikiskin' => 'Incompatible with MediaWiki 1.42',
-	'tweeki' => 'Incompatible with MediaWiki 1.42',
+	'snapwikiskin' => 'Incompatible with MediaWiki 1.42'
```

Comment:
Trailing commas should be kept. This prevents accidental syntax errors in the future when adding to the list.